### DISCUSSION TITLE
meshed output obj in meshlab is not like training result

### DISCUSSION THREAD

**Discussion by feiyilicare** [NVlabs/instant-ngp#357] [Q&A]
In gui, training result is correct,but meshed output obj in meshlab is not correct。
![Screenshot from 2022-03-18 17-31-25](https://user-images.githubusercontent.com/12027569/158978014-1b34bdb7-e170-4a9d-915d-70f9fcec5d64.png)

**Comment by Tom94** [comment]
Hi there, this codebase is not intended for high-quality mesh reconstruction. We provide the marching cubes feature as a proof-of-concept, but recommend using dedicated mesh-reconstruction methods to get higher-quality results. For example, the recently released https://github.com/NVlabs/nvdiffrec

**Reply by bonfry** [reply]
Hi @Tom94, what did I have to export from NeRF project for using it? Thank you
